### PR TITLE
mobifies scorpions

### PR DESCRIPTION
**Pull request by colossusqw**
[LABEL][AI][Rework]
## About the PR 
This PR:
This PR replaces the critter scorpions with a mob version. They will wander around and attack players and other mobs alike, and like their critter counterpart, can be tamed with food.

## Why's this needed? 
Mob > critter

### CODE REVIEW

File: code/mob/living/critter/small_animal.dm [hunk 1]
```diff
@@ -1691,6 +1697,42 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
...
+				if(prob(20))
+					friends += M
+					src.visible_message("[M] chitters happily at the \the [I], and seems a little friendlier with [M].")
+					src.emote("chitter")
+				else
+					src.visible_message("<span class='notice'>[src] hated \the [I] and bit [M]'s hand!</span>")
+					random_brute_damage(M, rand(6,12),1)
+					src.emote("snip")
+					M.emote("scream")
+			qdel(I)
```
Comment (amylizzle) [inline code/mob/living/critter/small_animal.dm:1720]: Maybe replace this with `I.Eat()`

**Review by amylizzle — CHANGES_REQUESTED** [review]
Code looks good. Only suggestion I have is to replace the `qdel()` call on the food with the standard mob eating behaviour.

**Comment by amylizzle** [comment]
Oh I meant in general, not specifically `I.Eat()`. You might wanna cast it to a snack and use the eat proc from that, with the appropriate arguments.

**Review by amylizzle — APPROVED** [review]

**Comment by pali6** [comment]
Does this need a changelog entry?

**Review by pali6 — APPROVED** [review]

**Comment by pali6** [comment]
Eh, I'm gonna with "doesn't".

**Pull request merged**